src/ua/griddynamics/Main.java

```diff
@@ -0,0 +1,46 @@
+package ua.griddynamics;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("How many mines do you want on the field? >");
+        int numberOfMines = scanner.nextInt();
+
+        char[][] hiddenField = new char[Game.HEIGHT][Game.WIDTH];
+        char[][] visibleField = new char[Game.HEIGHT][Game.WIDTH];
+        Game game = new Game(hiddenField, visibleField);
+
+        game.buildEmptyField();
+        System.out.println(game.getField(visibleField));
+        CoordinateChecker coordinateChecker = new CoordinateChecker();
+
+        boolean isMinesPlaced = false;
+        while (!game.isWin(numberOfMines)) {
+            String[] userAnswer = coordinateChecker.scanAndCheckUserInput();
+            Coordinate coordinate = coordinateChecker.getCoordinate(userAnswer, game);
+            if (!isMinesPlaced) {
+                game.placeMines(coordinate, numberOfMines);
+                game.numberGenerator();
```

Comment:
game.numberGenerator() - it should be a part of  placeMines functionality
We as a user don't have to know about this function at all